docs/AppService-dotnetcore.md

```diff
@@ -33,6 +33,9 @@ az webapp deployment source delete -g $He_App_RG -n $He_Name
 # this step usually takes 3-5 minutes
 az webapp deployment source config -g $He_App_RG -n $He_Name -u https://github.com/retaildevcrews/helium-csharp --branch main --manual-integration
 
+# set the app endpoint
+export He_App_Endpoint=https://${He_Name}.azurewebsites.net
```

Comment:
Is this valid?  I think [name].azurewebsites.net is only App Services based sites.